app/waf/src/sequence.ts

```diff
@@ -50,6 +50,7 @@ export class MySequence implements SequenceHandler {
   ) {}
 
   async handle(context: RequestContext) {
+    this.logger = factory.getLogger(context.name + ': api.call');
```

Comment:
Seems this line can be moved into `constructor`. And then remove the constructor's injection of `logger`.